boxoffice/views/admin.py

```diff
@@ -35,3 +38,26 @@ def jsonify_org(data):
 @render_with({'text/html': 'index.html', 'application/json': jsonify_org}, json=True)
 def org(organization):
     return dict(org=organization, title=organization.title)
+
+
+@app.route('/api/1/o/<org>/revenue', methods=['GET', 'OPTIONS'])
+@load_models(
+    (Organization, {'name': 'org'}, 'organization')
+    )
+def org_revenue(organization):
+    check_api_access(organization.details.get("access_token"))
+    if not request.args.get('year'):
+        return make_response(jsonify(status='error', message='Missing year.'), 400)
+
+    if not request.args.get('timezone'):
+        return make_response(jsonify(status='error', message='Missing timezone.'), 400)
+
+    user_timezone = request.args.get('timezone')
+    if user_timezone not in pytz.all_timezones:
```

Comment:
Use `pytz.common_timezones`. `all` includes some that PostgreSQL may not recognise.